WORKSPACE

```diff
@@ -113,3 +92,25 @@ http_archive(
 
 load("@bazel-common//:workspace_defs.bzl", "google_common_workspace_rules")
 google_common_workspace_rules()
+
+
+## For tests
+rules_play_routes_version = "6116a409bf1abcf78c4cfe1a8c55cfa01812bcc7"
+http_archive(
+  name = "io_bazel_rules_play_routes",
+  sha256 = "9849753b2a524bee018fb929b1f1d8698d0604bb3fc21e5294420a908ca6cb47",
+  strip_prefix = "rules_play_routes-{}".format(rules_play_routes_version),
+  type = "zip",
+  url = "https://github.com/lucidsoftware/rules_play_routes/archive/{}.zip".format(rules_play_routes_version),
+)
+
+load("@io_bazel_rules_play_routes//:workspace.bzl", "play_routes_repositories")
```

Comment:
Why don't we import the artifacts in this repo and not referencing from `rules_play_routes`?

Reply:
Good point; I'm actually going to remove most of the artifacts in this repo since they're not used directly